godotcord_store_manager.h

```diff
@@ -120,4 +33,4 @@ class GodotcordStoreManager : public Object {
 VARIANT_ENUM_CAST(GodotcordSKU::SkuType);

```

Comment:
I think it would be better to move the VARIANT_ENUM_CAST to the file where the enum is defined, so godotcord_store_data.h

Reply:
Oh alright!